errorz/multiple.go

```diff
@@ -31,3 +31,7 @@ func (e MultipleErrors) ToError() error {
 	}
 	return e
 }
+
+func ErrorListToError(errs []error) error {
```

Comment:
is this whole file needed? can it all just be replaced with 1.20+errors stuff now? for this one in partcular tho, why not just use `errors.Join` ?